src/packaging/resource/cassandra-reaper-cassandra-ssl.yaml

```diff
@@ -46,8 +47,11 @@ logging:
     io.dropwizard: WARN
     org.eclipse.jetty: WARN
   appenders:
-    - type: console
-      logFormat: "%-6level [%d] [%t] %logger{5} - %msg %n"
+    - type: file
+      logFormat: "%-6level [%t] %logger{5} - %msg %n"
+      currentLogFilename: reaper.log
+      archivedLogFilenamePattern: reaper-%d.log.gz
+      archivedFileCount: 99
```

Comment:
Shouldn't we specify a maximum size to control when rolling over will occur ?
Isn't 99 kind of a big number ?

Reply:
this rollover is daily.

when using `%d` it's daily, when using `%i` and maxFileSize it becomes size limited.

i just went for the simpler daily rotation approach here.